src/viur/core/skeleton.py

```diff
@@ -1220,8 +1264,11 @@ def fixDotNames(entity):
                 updateRelations(key, time() + 1, None)
 
         # Inform the custom DB Adapter of the changes made to the entry
-        if skel.customDatabaseAdapter:
-            skel.customDatabaseAdapter.updateEntry(db_obj, skel, change_list, is_add)
+        for adapter in utils.ensure_iterable(skel.customDatabaseAdapter):
+            db_obj = adapter.updateEntry(db_obj, skel, change_list, is_add)  # FIXME: DEPRECATED.
```

Comment:
It's just a hook.


https://github.com/viur-framework/viur-core/blob/41820476eb03252a604bc741ca3a815731940456/src/viur/core/skeleton.py#L651

```suggestion
            adapter.updateEntry(db_obj, skel, change_list, is_add)  # FIXME: DEPRECATED.
```